plugins/woocommerce-admin/client/core-profiler/actions/tracks.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import {
+	CoreProfilerStateMachineContext,
+	UserProfileEvent,
+	BusinessInfoEvent,
+} from '..';
+import { POSSIBLY_DEFAULT_STORE_NAMES } from '../pages/BusinessInfo';
+
+const recordTracksStepViewed = (
+	_ctx: unknown,
+	_evt: unknown,
```

Comment:
1. These parameters are named differently than for example, `recordTracksUserProfileCompleted`. 
2. We're not using them at the moment, could we just remove them to make it leaner?

Reply:
I think we could standardise on spelling them out, good call!

We can't remove them because it's (arg0, arg1, { action }) , so we need to keep the order in order to access arg2 (action meta object).